docs/README.md

```diff
@@ -1,46 +1,50 @@
+* **[Text Generation Inference Server (TGIS)](https://github.com/opendatahub-io/text-generation-inference)**: The serving backend that loads the models and provides the inference engine.
+* **[Caikit](https://github.com/opendatahub-io/caikit)**: A wrapper layer that handles the lifecycle of the TGIS process, provides the inference endpoints, and has modules to handle different model types.
+* **[Caikit-nlp](https://github.com/opendatahub-io/caikit-nlp)**: The Caikit module that handles Natural Language Processing (NLP)-style models.
+* **[KServe](https://github.com/opendatahub-io/kserve)**: A Kubernetes Custom Resource Definition that orchestrates model serving for all types of models. It includes serving runtimes that implement the loading of given types of model servers. KServe handles the lifecycle of the deployment object, storage access, and networking setup.
+* **[Service Mesh](https://docs.openshift.com/container-platform/4.13/service_mesh/v2x/ossm-architecture.html)** (istio): The service mesh networking layer that manages traffic flows and enforces access policies.
+* **[Serverless](https://docs.openshift.com/serverless/1.29/about/about-serverless.html)** (knative): A cloud-native development model that allows for serverless deployments of data models.
+
+## Architecture of the stack
+
+![KServe+Knative+Istio+Caikit_TGIS Diagram](https://github.com/opendatahub-io/caikit-tgis-serving/assets/8479010/7009b95d-0f6f-4f18-b0e6-355f360a5ad1)
 
 
 ## Installation
-### Prerequisites
-- Openshift Cluster 
-  - This doc is written based on a ROSA cluster and has been tested with an OSD cluster as well
-  - Many of the tasks in this tutorial require cluster-admin permission level (e.g., install operators, set service-mesh configuration, enable http2, etc)
-  - 4 CPU and 16 GB memory in a node for inferencing (can be adjusted in servingRuntime deployment)
-- CLI tools
-  - oc cli
 
+*Note:* The procedures for installing and deploying the Caikit-TGIS-serving image are based on a Red Hat OpenShift Service for AWS (ROSA) cluster. They have not been tested on an OpenShift Dedicated cluster.
+
+**Prerequisites**
+
+- Your cluster needs 4 CPU and 16 GB memory in a node for inferencing. (You can adjust these settings in the servingRuntime deployment.)
+- You need cluster administrator permissions for many of the procedures (such as, installing operators, setting service-mesh configuration, and enabling http2).
+- You have installed the OpenShift CLI (`oc`).
 
-### How to install
+**Procedure** 
 
-The following required operators will be installed as part of the KServe/Caikit/TGIS stack installation instructions.
+There are two ways to install the KServe/Caikit/TGIS stack:
+* [Step-by-step command installation](/demo/kserve/install-manual.md)
+* [Script-based installation](/demo/kserve/scripts/README.md)
+
+**Installed operators**
+
+The following operators are installed when you install the KServe/Caikit/TGIS stack:
```

Comment:
@MelissaFlinn Could we mention here that these operators are pre-requisites to KServe/Caikit/TGIS?
A little background on it: currently we install these operators as part of our script since they are required for the KServe/Caikit/TGIS stack, but in the future we would like to just provide links for installation and have some logic in ODH/RHODS that will check that these are present on the cluster.
@Jooho please correct me if I am wrong.